examples/add_line_type.js

```diff
@@ -1,35 +1,20 @@
-const StreamableDrawing = require('../src/StreamableDrawing');
-const Drawing = require('./../src/Drawing');
-const fs = require('fs');
+const NodeJsDrawing = require('../src/NodeJsDrawing');
+const BrowserFriendlyDrawing = require('../src/BrowserFriendlyDrawing');
+const mainModule = require('./mainModule');
```

Comment:
Very descriptive names haha 👍

Reply:
Sorry, I was struggling to differentiate them in better ways 🤣